docs/development/plugn-triggers.md

```diff
@@ -1,15 +1,15 @@
-# Pluginhooks
+# Plugn triggers
```

Comment:
Basically a page saying the old `pluginhooks.md` docs can now be found at `plugn-triggers.md`. To be honest, I feel as though we should keep the old filename with the new content.